features/FEATURE_BLE/targets/TARGET_CORDIO/CordioPalSecurityManager.h

```diff
@@ -326,6 +323,15 @@ class CordioSecurityManager : public ::ble::pal::SecurityManager {
 private:
     bool _use_default_passkey;
     passkey_num_t _default_passkey;
+    bool _lesc_keys_generated;
+    uint8_t _public_key_x[SEC_ECC_KEY_LEN];
+    bool _peer_oob_present;
+    address_t _peer_oob_address;
+    oob_lesc_value_t _peer_oob_random;
+    oob_confirm_t _peer_oob_confirm;
+    bool _own_oob_present;
```

Comment:
Do we need to keep this state on pal side and user side ?

Reply:
there is a pal function, is oob present, if the generic needs to know it will use this to ask